app/src/components/ProjectRegistration/TransactionSummaryStep.svelte

```diff
@@ -55,15 +55,13 @@
     <Avatar
       variant="project"
       imageUrl="https://avatars.dicebear.com/v2/jdenticon/project.svg"
-      title={name} />
+      title={projectName} />
   </div>
 </TxRow>
 
 <TxRow disabled={true} style="border-radius: 0; margin: -1px 0 -1px 0">
   <div slot="left">
-    <Title style="color: var(--color-darkgray);">
-      Transaction Fee
-    </Title>
+    <Title style="color: var(--color-darkgray);">Transaction Fee</Title>
```

Comment:
Unrelated formatting change?

Reply:
These must have been caused by the VSCode svelte support plugin. Sorry for the noise, it's really annoying.

Reply:
✅

Reply:
Actually, the linter caused these changes and requires them to build. Reverted and brought them back.